server/settings/publish_plugins.py

```diff
@@ -43,6 +43,13 @@ class ValidateSceneSettingsPlugin(BaseSettingsModel):
     )
 
 
+class ValidateInstancePlugin(BaseSettingsModel):
+    """Validate if instance folder is the current folder."""
+    enabled: bool = True
```

Comment:
For sake of consistency in the codebase, could you add the defaults here too: https://github.com/Tilix4/ayon-harmony/blob/fix-validate_instance_blocking_flexibility/server/settings/main.py#L62

Reply:
done